addons/mail/static/src/js/systray/systray_activity_menu.js

```diff
@@ -127,6 +127,20 @@ var ActivityMenu = Widget.extend({
         }
     },
 
+    /**
+     * Return views to display when coming from systray depending on the model.
+     * @private
+     */
+    getViewsList: function(model){
+        var views = [[false, 'kanban'], [false, 'list'], [false, 'form']];
+        switch(model){
```

Comment:
js guidelines: https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines
```suggestion
        switch (model) {
```